students/yechan/westagram/users/views.py

```diff
@@ -36,13 +38,21 @@ class SingInView(View):
     def post(self, request):
         try:
             data     = json.loads(request.body)
-            email    = data['email']
-            password = data['password']
+            user     = User.objects.get(email=data['email'])
+
+            if not bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
+                return JsonResponse({'message':'INVALID_PASSWORD'},status=401)
```

Comment:
```suggestion
            if not bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
                return JsonResponse({'message':'INVALID_USER'},status=401)
```
로그인 관련 에러처리에서 email이 틀렸는지, 비밀번호가 틀렸는지 명확하게 안 나타내는 것이 좋습니다~